media-tv/kodi/kodi-17.3.ebuild

```diff
@@ -0,0 +1,279 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+# Does not work with py3 here
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite"
+
+inherit eutils linux-info python-single-r1 cmake-utils autotools
```

Comment:
diff --git a/media-tv/kodi/kodi-17.2.ebuild b/media-tv/kodi/kodi-17.2.ebuild
index 94c5a55bfa..babff0372d 100644
--- a/media-tv/kodi/kodi-17.2.ebuild
+++ b/media-tv/kodi/kodi-17.2.ebuild
@@ -7,7 +7,7 @@ EAPI=6
 PYTHON_COMPAT=( python2_7 )
 PYTHON_REQ_USE="sqlite"
 
-inherit eutils linux-info python-single-r1 cmake-utils autotools
+inherit eutils linux-info python-single-r1 cmake-utils autotools pax-utils
 
 LIBDVDCSS_COMMIT="2f12236bc1c92f73c21e973363f79eb300de603f"
 LIBDVDREAD_COMMIT="17d99db97e7b8f23077b342369d3c22a6250affd"
@@ -254,6 +254,9 @@ src_compile() {
 
 src_install() {
 	cmake-utils_src_install
+
+	pax-mark E "${ED%/}"/usr/$(get_libdir)/${PN}/${PN}.bin
+
 	rm "${ED%/}"/usr/share/doc/*/{LICENSE.GPL,copying.txt}* || die
 
 	newicon media/icon48x48.png kodi.png
That should point you in the right direction. I have not tested to ensure accuracy, just something to help get you started. I do not we need pax-mark E for sure tho.